.travis.yml

```
@@ -29,11 +29,8 @@ matrix:
   allow_failures:
     - go: master
 
-before_install:
-  - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-
 install:
-  - dep ensure -v
+  - go mod vendor
```

```suggestion
  - go mod vendor -v
```

Do we want to enable verbose mode?:)